### PR TITLE
add more tests to the `polars` plugin

### PR DESCRIPTION
**Pull request by maxim-uvarov**
# Description

I added some more tests to our mighty `polars` ~~, yet I don't know how to add expected results in some of them. I would like to ask for help.~~

~~My experiments are in the last commit: [polars: experiments](https://github.com/nushell/nushell/commit/f7e5e72019ee55b0e6745c1dc405c9a37ced67d1). Without those experiments `cargo test` goes well.~~
 
UPD. I moved out my unsuccessful test experiments into a separate [branch](https://github.com/maxim-uvarov/nushell/blob/polars-tests-broken2/). So, this branch seems ready for a merge.

@ayax79, maybe you'll find time for me please? It's not urgent for sure. 

P.S. I'm very new to git. Please feel free to give me any suggestions on how I should use it better

### CODE REVIEW

File: crates/nu_plugin_polars/src/dataframe/eager/sample.rs
Comment (ayax79) [inline crates/nu_plugin_polars/src/dataframe/eager/sample.rs:1]: There needs to be a section to execute the tests. It should look something like this:

```rust
#[cfg(test)]
mod test {
    use super::*;
    use crate::test::test_polars_plugin_command;

    #[test]
    fn test_examples() -> Result<(), ShellError> {
        test_polars_plugin_command(&SampleDF)
    }
}
```

File: crates/nu_plugin_polars/src/dataframe/series/date/as_date.rs
Comment (ayax79) [inline crates/nu_plugin_polars/src/dataframe/series/date/as_date.rs:1]: This file also needs to have the test harness code added (see previous comment).

**Review by ayax79 — CHANGES_REQUESTED** [review]
Looks good, besides the couple of mentioned issues.

File: crates/nu_plugin_polars/src/dataframe/eager/sample.rs
Comment (maxim-uvarov) [reply crates/nu_plugin_polars/src/dataframe/eager/sample.rs:1]: thanks! done

File: crates/nu_plugin_polars/src/dataframe/series/date/as_date.rs
Comment (maxim-uvarov) [reply crates/nu_plugin_polars/src/dataframe/series/date/as_date.rs:1]: Thanks! done

**Comment by fdncred** [comment]
Thanks!

**Pull request merged**